mbbsd/mail.c

```diff
@@ -1195,6 +1195,105 @@ mail_all(void)
 }
 
 int
+get_account_sysop(struct Vector *namelist)
+{
+    FILE *fp;
+    if ( (fp = fopen(FN_MAIL_ACCOUNT_SYSOP, "r")) != NULL) {
+        char line[STRLEN+1];
+        while(fgets(line, sizeof(line), fp)) {
+            chomp(line);
+
+            char userid[IDLEN+1];
```

Comment:
I'd prefer to not declaring variables after code.

Try to move this some where after char line.